src/main/resources/tags/info/bungee.tag

```diff
@@ -3,6 +3,10 @@ aliases: bungeecord, velocity
 
 ---
 
-The Geyser and Floodgate BungeeCord plugins only need to be installed on the BungeeCord proxy unless you intend to use the Floodgate API. That way, you need Floodgate on the "backend" servers too.
-Installing Floodgate on the backend servers also makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying.
-Run `!!api` in <#613194762249437245> for more information regarding the Floodgate API.
+For the most basic installation, the Geyser and Floodgate Bungeecord or Velocity plugins only need to be installed on the proxy.
+
+For more advanced/complete features, you may also install Floodgate on the backend servers:
+***-*** It makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying (once they're uploaded to the skin server).
+***-*** If you intend to use certain plugins such as [BedrockPlayerManager](https://www.spigotmc.org/resources/bedrock-player-managment-for-floodgate-geyser.82278/)
+***-*** If you wish to use the API for your own purposes. You may run `!!api` in <#613194762249437245> for more information regarding the Floodgate API.
+See the [wiki](https://github.com/GeyserMC/Floodgate/wiki/Setup-and-Usage#installing-floodgate-also-on-spigot-servers-behind-bungeecord-or-velocity) for more info on installing Floodgate on your backends.
```

Comment:
To my knowledge, you can't embed links like this. I haven't tested, though.

Reply:
It should

e.g.
https://github.com/GeyserMC/GeyserDiscordBot/blob/master/src/main/resources/tags/links/providers.tag
https://github.com/GeyserMC/GeyserDiscordBot/blob/master/src/main/resources/tags/info/developerguide.tag

Those examples don't have anchor links but I don't see why those wouldn't work

Reply:
it would show them like shown here in this image
![image](https://user-images.githubusercontent.com/61922763/134152952-c74db246-48dc-423d-9aa7-94a5dc500189.png)